redis/src/cluster_async/mod.rs

```diff
@@ -383,6 +402,9 @@ pub(crate) struct InnerCore<C> {
     push_sender: Option<mpsc::UnboundedSender<PushInfo>>,
     subscriptions_by_address: RwLock<HashMap<String, PubSubSubscriptionInfo>>,
     unassigned_subscriptions: RwLock<PubSubSubscriptionInfo>,
+    disconnect_notifier: Option<Box<dyn DisconnectNotifier>>,
+    #[cfg(feature = "tokio-comp")]
+    tokio_notify: Arc<Notify>,
```

Comment:
Yep, that was my original try, but got :

```
102 |     async fn notified(&mut self);
    |     -----^^^^^^^^^^^^^^^^^^^^^^^^
    |     |
    |     `async` because of this
    |
    = note: `async` trait functions are not currently supported
```
There might be some crates that do it, but i dont want to go down this rabbit hole.
Do you know how to do it?

Reply:
done

Reply:
```
#[async_trait::async_trait]
```